src/data/form/Interest.ts

```diff
@@ -1,7 +1,45 @@
 import data from "@/data/Config";
 import { SOURCES } from "./Information";
 
-export const FIELDS = {
+type descriptionField = {
```

Comment:
these are already implemented in another file, check the other form files

Reply:
![image](https://github.com/user-attachments/assets/aee132b9-955e-40e3-8329-9d8c2e9d7d19)
![image](https://github.com/user-attachments/assets/dc5da910-4c81-422c-b2e4-db89e34be948)
![image](https://github.com/user-attachments/assets/ed4dc6eb-a663-4298-819d-c87052be917d)
![image](https://github.com/user-attachments/assets/fd876bcd-b351-438a-ab3c-64ea59376a94)
![image](https://github.com/user-attachments/assets/4c878b10-27d2-4419-9d2d-b8f94a15ce9b)
Used types imported from forms.ts
added missing properties
moved ATTRIBUTE constant to above + added attribute interface